recipes/numactl-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: numactl-cos6-x86_64
+  version: 2.0.9
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/numactl-2.0.9-2.el6.x86_64.rpm
+    sha256: 70e58c3b01ce62d90ef0e3d9f1c0d1d3efaa4d237b3b821ee7d1e791553d82a7
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/numactl-2.0.9-2.el6.src.rpm
+    folder: source
+
+build:
+  number: 0
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+
+###########################################################
+# Commented out to avoid a conda-build bug.               #
+# xref: https://github.com/conda/conda-build/issues/3553  #
+###########################################################
+#
+#test:
+#  commands:
+#    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
```

Comment:
Currently adding a `test` section results in issue ( https://github.com/conda/conda-build/issues/3553 ). So it is commented for now.

Reply:
Have since added a workaround so we can add the tests back.